test.c

```diff
@@ -1,12 +1,66 @@
-
 #include "common.h"
 
+void example1(void);
+void testStore(void);
+void testRetrieve(void);
+void testParseGrammer(void);
+void assert(bool value, char* errorMsg);
+
+
+struct ParseTree* parseGrammer(char* sql) {
```

Comment:
These three functions are placeholders so it compiles. They don't do anything, and as a result all of these tests fail.